iup/__init__.py

```diff
@@ -8,33 +8,18 @@
 import re
 
 
-class ValidateData(pl.DataFrame, metaclass=abc.ABCMeta):
+class ValidatedData(pl.DataFrame):
```

Comment:
Although it doesn't affect how the code works, "ValidatedData" is a weird name to me. What do you think about changing the name of the parent class to "Uptake" and the names of the child classes to "IncidentData," "CumulativeData," "QuantileForecast," "PointForecast," and "SampleForecast" (the last three already being correct)? This makes more sense to me - it's all uptake, but the distinction is between data vs. forecasts. But of course I'm happy to do this in a separate PR, since it's just a lot of nuisance work!

Reply:
"ValidatedData" indicates that all the classes are the data that is validated, where the data means generic data set. While differing 'data' from 'forecast' indicates it is observed data. This makes sense too. I will change to "ValidatedUptake", emphasizing all the classes have been validated when initialized.

Reply:
Changed in 4e77134bb18944b64e1e91c9f6b544132bae8c73